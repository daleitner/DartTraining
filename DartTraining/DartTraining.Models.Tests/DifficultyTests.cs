using ApprovalTests;
using ApprovalTests.Reporters;
using Base;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace DartTraining.Models.Tests
{
	[TestClass]
	[UseReporter(typeof(DiffReporter))]
	public class DifficultyTests
	{
		protected Difficulty SUT;

		[TestInitialize]
		public void Setup()
		{
			IdGenerator.PrepareGeneratorForTest(0);
		}

		[TestMethod]
		public void WhenCreateNewDifficulty_ThenVerifySUT()
		{
			var defaultLevel = 1;
			SUT = new Difficulty(defaultLevel);
			Approvals.Verify(SUT.ToPrettyString());
		}
	}
}
