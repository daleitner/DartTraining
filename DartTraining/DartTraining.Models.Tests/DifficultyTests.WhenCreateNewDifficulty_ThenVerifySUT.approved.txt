Δ Difficulty
┌──────────────────────────────────────┬───────────────┬─────────┐
│ [Id]                                 │ [DisplayName] │ [Level] │
├──────────────────────────────────────┼───────────────┼─────────┤
│ 00000000-0000-0000-0000-000000000000 │               │ 1       │
└──────────────────────────────────────┴───────────────┴─────────┘
